lib/plugins/simple_inventory.js

```diff
@@ -109,7 +109,7 @@ function inject (bot) {
       return
     }
 
-    if (destSlot >= QUICK_BAR_START && sourceSlot >= QUICK_BAR_START) {
+    if (destSlot >= QUICK_BAR_START > (QUICK_BAR_START + QUICK_BAR_COUNT) && sourceSlot >= QUICK_BAR_START > (QUICK_BAR_START + QUICK_BAR_COUNT)) {
```

Comment:
`QUICK_BAR_START > (QUICK_BAR_START + QUICK_BAR_COUNT)` is always false, I suspect you meant to compare with destSlot and sourceSlot instead of QUICK_BAR_START ?

Reply:
Yes. I wanted to compare it with destSlot and sourceSlot so it is within QUICK_BAR_START and QUICK_BAR_START + QUICK_BAR_COUNT.